pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -142,6 +154,69 @@ func (e *cloudWatchExecutor) getRGTAClient(region string) (resourcegroupstagging
 	return e.rgtaClient, nil
 }
 
+func (e *cloudWatchExecutor) getServiceQuotasClient(region string) (servicequotasiface.ServiceQuotasAPI, error) {
+	sess, err := e.newSession(region)
+	if err != nil {
+		return nil, err
+	}
+
+	return newQuotasClient(sess), nil
+}
+
+func (e *cloudWatchExecutor) getQueue(region string) (chan bool, error) {
+	e.queueLock.Lock()
+	defer e.queueLock.Unlock()
+
+	if queue, ok := e.queuesByRegion[region]; ok {
+		return queue, nil
+	}
+
+	concurrentQueriesQuota, err := e.fetchConcurrentQueriesQuota(region)
+	if err != nil {
+		plog.Info("Could not fetch quota")
```

Comment:
Why not return an error here? The value of concurrentQueriesQuota should be inconsistent in this case anyhow.

Reply:
Because rather than having everything fail it seemed to make more sense to just use a hardcoded default (4 in this case), something that's possible since certain permissions are required on the AWS end to retrieve the quota values.